static/docs/user-guide/data-sharing/remote-storage.md

```diff
@@ -0,0 +1,193 @@
+# Sharing Data Through a Remote DVC Storage
```

Comment:
it is strange to see that it contains only examples, no explanation of what's happening whatsoever. I would expect it too explain remotes way better - this is a primary purpose of this.

SSH example is too complicated - 

- the same dir for git and dvc on remote is too specific, very uncommon and distracts a lot
- name of the remote storage should not use cache in it

Reply:
> it is strange to see that it contains only examples, no explanation of what's happening whatsoever

It is a modified version of this page: https://dvc.org/doc/use-cases/sharing-data-and-model-files
That one does not have much explanations either and things are explained mostly by the example.

Actually I don't find it feasible to explain a solution without using at least a few DVC commands, and for those commands to make sense they have to be used in the context of an example. So, the description mainly describes the situation, and the solution is described by the examples. The hope is that once the reader has understood the solution he can generalize and adopt it for his own case.

> I would expect it too explain remotes way better - this is a primary purpose of this.

I am planning to explain the details of the remotes (and their types) on another section. This section is about data sharing scenarios, so let's just refer to the remote details, but not include them here.

> the same dir for git and dvc on remote is too specific, very uncommon and distracts a lot

Yes, the Git repository is usually located on GitHub. But this is just an example, an assumption to keep things simple and interactive.

> name of the remote storage should not use cache in it

I tried to keep the analogy with Git. In Git a central bare repository is usually named `project.git`. So, a central DVC storage/cache is name `project.cache`.

Reply:
> I am planning to explain the details of the remotes (and their types) on another section. This section is about data sharing scenarios, so let's just refer to the remote details, but not include them here.

I don't think we need both that was point. I'm confused why do we need both. I think the remote section is enough. There should be some "DVC workflow" section that explains from a high level perspective the workflow.

Reply:
> I don't think we need both

I don't think that these UG pages:
https://dvc-org-pr-807.herokuapp.com/doc/user-guide/external-data  (from another PR)
can be merged with the pages of this PR. I think they should be separate sections.